### PR TITLE
Upgrade cadl ranch to the latest

### PR DESCRIPTION
**Pull request by MaryGao**
Upgrade cadl ranch to the latest

fixes https://github.com/Azure/autorest.typescript/issues/1534

### CODE REVIEW

File: packages/cadl-rlc-emitter/test/unit/generate/docs.spec.ts [hunk 1]
```diff
@@ -162,4 +166,39 @@ describe("Doc generation testing", () => {
...
+            get(options?: ReadParameters): StreamableMethod<Read204Response>;
+        }
+  
+        export interface Routes {
+            /** Resource for '/' has methods for the following verbs: get */
+            (path: "/"): Read;
+        }
+        
+        export type testClient = Client & {
+                path: Routes;
```
Comment (qiaozha) [inline packages/cadl-rlc-emitter/test/unit/generate/docs.spec.ts:198]: Can we use prettier and compare with the result after prettier ?

File: packages/cadl-rlc-emitter/test/integration/generated/models/propertyTypes/package.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
 {
-  "name": "@msinternal/enum-properties",
+  "name": "@msinternal/property-types",
   "sdk-type": "client",
```
Comment (MaryGao) [inline packages/cadl-rlc-emitter/test/integration/generated/models/propertyTypes/package.json:3]: As this [pr ](https://github.com/Azure/cadl-ranch/pull/86)commented the removed cases will be replaced with in new one `property-types`

Comment (qiaozha) [reply packages/cadl-rlc-emitter/test/integration/generated/models/propertyTypes/package.json:3]: ok

File: packages/cadl-rlc-emitter/test/unit/generate/docs.spec.ts (hunk 1 above)
Comment (MaryGao) [reply packages/cadl-rlc-emitter/test/unit/generate/docs.spec.ts:201]: updated

**Review by qiaozha — APPROVED** [review]

**Pull request merged**